test/__init__.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import collections
+import json
+import logging
+import os
+import tarfile
+import time
+
+import boto3
+import pytest
+import sagemaker
+import six
+
+import sagemaker_containers as smc
+
+DEFAULT_REGION = 'us-west-2'
+
+DEFAULT_CONFIG = dict(ContentType="application/x-numpy", TrainingInputMode="File",
```

Comment:
Why the preference for dict() rather than {}?

Reply:
I interchange the use of dict and {}, they have different advantages. Dict usually are shorter and easier to read. {} can work with dynamic keys. Both are valid syntax.

I will change all the " for ', since we agreed in our sintax guide